doc/guide/build_windows.hpp

```diff
@@ -9,8 +9,13 @@
 
 @section build_windows_intro Introduction
 
-This tutorial will show you how to build mlpack for Windows from source, so you can
-later create your own C++ applications. Before you try building mlpack, you may
+This tutorial will how you how to build mlpack for Windows from source, so
+you can later create your own C++ applications, using two different ways:
+
+  - Using CMake to generate an intermeditate Visual Studio solution (`.sln`).
+  - Use Visual Studio's CMake integration to directly build from the `CMakeLists`.
```

Comment:
If you want we could use a Doxygen reference here:

```
@ref build_visual_studio_cmake_integration "Use Visual Studio's ..."
```

I think that will work and would allow a reader to quickly jump to the section. :+1:

Reply:
Applied this suggestion as well.